test/config-next/sa.json

```diff
@@ -36,6 +36,14 @@
             "wfe.boulder"
           ]
         },
+        "sa.StorageAuthorityReadOnly": {
+          "clientNames": [
+            "crl-updater.boulder",
```

Comment:
Why does the CRL updater talk to the readonly instance?

Reply:
Because the only thing it does is read from the CertificateStatus table. You're suggesting that it could/should talk to the read-write service, because it's not in the EFN and it may want to update a serial-to-shard mapping in the future?